tag.php

```diff
@@ -0,0 +1,60 @@
+<?php
+	get_header();
+
+	$posts = get_posts( array(
+		'numberposts' => 10,
+		'tag'         => get_queried_object()->slug
+	) );
+
+	if( isset( $posts ) ) {
+		$first_post = array_shift( $posts );
+	}
+
+	if( class_exists( 'UCF_Events_Config' ) ) {
+		$events_feed_url = UCF_Events_Config::get_option_or_default( 'feed_url' );
+		$all_events_link = str_replace( 'feed.json', '', $events_feed_url );
+	}
+?>
+
+<div class="container mt-4 mb-5 pb-sm-4">
+	<div class="row">
+		<div class="col-md-8">
+			<?php if ( $first_post ): ?>
+				<?php echo today_display_feature_vertical( $first_post, array( 'layout__type' => 'primary' ) ); ?>
+			<?php endif; ?>
+
+			<?php if ( $posts ): ?>
+				<div class="row">
+					<?php foreach ( $posts as $post ): ?>
+						<div class="col-lg-4">
+							<?php echo today_display_feature_vertical( $post ); ?>
+						</div>
+					<?php endforeach; ?>
+				</div>
+			<?php endif; ?>
+
+			<?php if ( ! $first_post && ! $posts ): ?>
+				<p>No results found.</p>
+			<?php endif; ?>
+		</div>
+		<div class="col-md-4">
```

Comment:
Related to the comment above, changing this to `col-lg-4` would help with the content having some more space in the -md display. 
```suggestion
		<div class="col-lg-4">
```